package.json

```diff
@@ -42,6 +42,7 @@
     "acorn": "^8.0.4",
     "archiver": "^5.1.0",
     "chart.js": "~2.9.4",
+    "chartjs-test-utils": "^0.1.3",
```

Comment:
We can probably use 0.2.0 in this PR so `chartjs-next` will benefit from the async trigger mouse event.